### PR TITLE
fix(clerk-js): Organization Profile polishing

### PR DESCRIPTION
**Pull request by octoper**
## Description

Polishing `<OrganizationProfile/>` component.

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [X] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [X] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [X] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [X] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by anagstef** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @anagstef, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-9byf3wxpa.clerkstage.dev) | Dec 19, 2023 08:26 PM |

File: packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx [hunk 1]
```diff
@@ -22,11 +22,6 @@ import { VerifyDomainScreen } from './VerifyDomainScreen';
 type DomainListProps = GetDomainsParams & {
   verificationStatus?: OrganizationDomainVerificationStatus;
   enrollmentMode?: OrganizationEnrollmentMode;
-  /**
-   * Enables internal links to navigate to the correct page
-   * based on when this component is used
-   */
-  redirectSubPath: 'organization-settings/domain' | 'domain';
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx:29]: @octoper Is this ok to be removed?

Comment (octoper) [reply packages/clerk-js/src/ui/components/OrganizationProfile/DomainList.tsx:29]: Yeah should be safe to remove that now that everything is inlined.

**Review by anagstef — APPROVED** [review]

**Pull request merged**